LeaderboardBackend.Test/Services/AuthServiceTests.cs

```diff
@@ -14,38 +13,37 @@ public class AuthServiceTests
 	[Test]
 	public void GenerateJSONWebToken_CreatesTokenWithConfig()
 	{
-		string key = "testkeythatsatisfiesthecharacterminimum";
-		string issuer = "leaderboards.gg";
+		var key = "testkeythatsatisfiesthecharacterminimum";
```

Comment:
I don't personally like it, but I won't fight using var for literal types if people are a fan of it.

Reply:
Addressed in 76b68bfb5c7596c0a5c7d26ac909dfea1e14d13d (`AuthServiceTests.cs` [`R17-19`](https://github.com/leaderboardsgg/leaderboard-backend-poc/commit/76b68bfb5c7596c0a5c7d26ac909dfea1e14d13d#diff-97760215483327af6ee209635916664ea48f33a8bb5fd7bcc282ce3b9a1d4c1eR17-R19)).